chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts

```diff
@@ -104,7 +106,8 @@ export const initWebChatComposer = (props: ILiveChatWidgetProps, state: ILiveCha
         markdownRenderers.forEach((renderer) => {
             text = renderer.render(text);
         });
-
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        text = DOMPurify.sanitize(text);
```

Comment:
Do you have a test src for this?

Reply:
https://elastorageredmond.blob.core.windows.net/elopez-dev/dev/scripts/LiveChatBootstrapper.js